concent_api/verifier/tasks.py

```diff
@@ -188,7 +189,12 @@ def blender_verification_order(
         completed_process = run_blender(
             scene_file,
             output_format,
+            blender_crop_script,
         )
+
+        # Remove the script after rendering to make sure that the next task does not reuse it.
+        delete_file(blender_crop_script)
```

Comment:
One is in `try`, other is in `except` block, and `except` block does `return` (`try` is doing to if non-zero exit code is returned from subprocess) so it can't be done after whole `try/except`.

Reply:
It can. You could use `try ... finally`. Actually you must use `try ... finally` because otherwise the file is not deleted if an exception other than `SubprocessError` is raised.

Reply:
:+1: done